sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventHubConsumerAsyncClientIntegrationTest.java

```diff
@@ -439,30 +439,43 @@ public void receivesMultiplePartitions() {
             () -> logger.info("Completed"));
 
         // Act & Assert
+        AtomicInteger removedPartitionsNumber = new AtomicInteger(expectedNumber);
         try {
             StepVerifier.create(consumer.receive(false)
                 .filter(x -> TestUtils.isMatchingEvent(x.getData(), MESSAGE_TRACKING_ID))
                 .take(expectedNumber))
-                .assertNext(event -> assertPartitionEvent(event, producer.getEventHubName(), allPartitions,
```

Comment:
The intention of this test was that we would receive from all 6 partitions. The test here is updated so that it receives 6 times, it doesn't discern between what partitions were received from.

Reply:
Since this test case sets a limit on the number of received pieces, in order to eliminate the interference of other test case data on the verification results, so modified the filter conditions of the receive event data.

Reply:
@conniey Any ideas?